src/views/DatasetView.vue

```diff
@@ -143,7 +142,7 @@ export default defineComponent({
                 msg: "oarec",
                 icon: "mdi-book-search",
               });
-            } else if (link.rel === "collection" && hasObs) {
+            } else if (link.rel === "collection" && hasSynop) {
```

Comment:
please remove `&& hasSynop`, the "observations"-button should be available for any link to a collection ...